style1.css

```diff
@@ -0,0 +1,389 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,500;1,400&display=swap');
+
+*,
+::after,
+::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  position: absolute;
+  width: 375px;
+  height: 1263px;
+  left: 1px;
+  top: 1px;
+  background: #fff;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background: #e5e5e5;
+}
+
+a {
+  text-decoration: none;
+}
+
+#navigation {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding: 25px;
+  background-color: #fff;
+}
+
+.logo {
+  text-transform: capitalize;
+  width: 76px;
+  height: 20px;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.hamburger {
+  width: 18px;
+  height: 12px;
+  color: #6070ff;
+  cursor: pointer;
+}
+
+.hero-section {
+  background: #fff;
+  padding: 1em 1em;
+  background-image: url(/images1/header-shapes\ mobile@2x.svg);
+  background-size: 100% auto;
+  background-position: center;
+  border-bottom-left-radius: 90px;
+  background-repeat: no-repeat;
+  min-height: 587.15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+#name {
+  width: 85%;
+  height: 150px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-top: 46px;
+}
+
+.details {
+  width: 95%;
+  height: 146px;
+  margin-top: 1em;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.connect {
+  text-transform: capitalize;
+  padding-top: 1.5em;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.connect:hover {
+  color: #344563;
+}
+
+.socials {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  width: 170.17px;
+  height: 20px;
+  margin-top: 1.5em;
+}
+
+#work-section {
+  width: 95%;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-flow: row;
+  justify-items: center;
+  align-items: center;
+  margin-top: 46px;
+  padding: 1em 1em;
+}
+
+.project {
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin: 50px auto;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.work-img img {
+  width: 100%;
+}
+
+.work-description {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
```

Comment:
- [ ] You did a good job but still, you can do better by improving your design. Kindly fix the UI issues that I have pointed out in your work, please try to give the gap on the parent container instead of using `margin` on each and every child element. Please have a look at the below-attached table of design comparison so that you can understand where exactly is the issue. 

<table>
<tr>
<th>
Figma Design
</th>
<th>
Your Design
</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-28 17-13-37](https://user-images.githubusercontent.com/66476329/235190146-4ff6c8ba-d62e-47f7-bf05-d678f91383b8.png)

</td>
<td>

![Screenshot from 2023-04-28 20-28-32](https://user-images.githubusercontent.com/66476329/235190192-6737245f-0034-4326-a56f-49ea4fcbb6f5.png)

</td>
</tr>

</table>

Reply:
- [ ] Please kindly ensure to effect the changes listed earlier. Our ability to pay attention to details as developers will really help us to be more skillful. 👍 **Kindly implement appropriate spacing and the counter beside the text**

> |     | Given template                                | Your Design                                   |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     | <img width="235" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235226238-379cc02d-29f1-42f0-94cd-337d6894bab9.png"> | <img width="248" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/235226832-aaa72124-4ab0-41ca-9476-fd12deb6dea5.png"> |